content/news/033/index.md

```diff
@@ -31,8 +31,7 @@ Feel free to send PRs about your own projects!
 [Rust]: https://rust-lang.org
 [join]: https://github.com/rust-gamedev/wg#join-the-fun
 
-- [Rust GameDev Meetup](#rust-gamedev-meetup)
-- [Rust Graphics Meetup](#rust-graphics-meetup)
+- [IRL Events](#irl-events)
```

Comment:
Thoughts on what this could be titled better as?

I'm thinking "Live events", or maybe "Bulletin board"

Reply:
Originally, I intended to use custom top-level sections for "official announcements" from the Gamedev WG (events, our [blog posts](https://gamedev.rs/blog), podcast episodes, talk recordings, etc), but in practice "non-official" important stuff is posted like this as well. Maybe, just "Announcements" isn't that bad?

Reply:
I like 'Announcements' 👍

Reply:
Nice, I like it :)